android/src/main/java/com/magicleap/magicscript/scene/nodes/prism/Prism.kt

```diff
@@ -170,7 +176,17 @@ class Prism(
     private fun buildContainer(transformationSystem: TransformationSystem, size: Vector3) {
         // detach old container if exists
         detachContainer()
-        container = PrismContentNode(transformationSystem, cubeBuilder, size).also { container ->
+
+        val packageName = appInfoProvider.getPackageName()
+        val cornerModelPath = Uri.parse("android.resource://$packageName/raw/prism_corner")
```

Comment:
Will this path be the same for debug and release? If not, maybe you can use the FileProvider class we use for subtitles and audio files

Reply:
Yes, the path will be always the same, because this is our local library file.

Reply:
Ok, no further questions :)